src/Controls/src/Core/BindableObject.cs

```diff
@@ -28,6 +29,9 @@ public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceH
 		public IDispatcher Dispatcher =>
 			_dispatcher ??= this.FindDispatcher();
 
+
+		IElement Element => _element ??= this as IElement;
```

Comment:
Is this really needed? It seems to just be used in once place and I am not sure a cast is worth the slightly confusing code.